### DISCUSSION TITLE
A question about leaderboard

### DISCUSSION THREAD

**Discussion by imrooki** [autogluon/autogluon#2091] [Q&A] [answered]
1. My model's leaderboard missing some data of score_val and score_test, is there anyone who can help explain the reason?

**Comment by easyautoml** [comment, marked as answer]
I guess that after trained, the finnal model will be retrain on both train and evaluation set. So before model is refited, that model will have score_val, and after refit, the refit model doens't have score_val any more, only score_test remain. 
Flows of training model might as bellow : 
- Model A : trained on train set, evaluation on evaluation set -> We have model A with score_eval and best hyper parameter also
- Model A_FULL : trained on both train + eval with hyper param found at last step, then evaluate on test set -> We have model A_FULL with score_test.

Leader board migh look like this
Model A---------score_eval : 123--------score_test : NULL
Model A_FULL--score_eval :  NULL------score_test : 123

**Reply by imrooki** [reply]
Thanks Sir, you have solved my confusion.
